app/src/main/java/org/gdg/frisbee/android/common/GdgNavDrawerActivity.java

```diff
@@ -401,7 +400,7 @@ private void maybeUpdateChapterImage() {
                     @Override
                     public Person doInBackground(String... params) {
 
-                        return PlusPersonDownloader.getPersonSync(params[0]);
+                        return App.getInstance().getPlusApi().getPerson(params[0]);
```

Comment:
For this PR, let's keep the changes small and refactor all these AsyncTask in the next step.